.devcontainer/devcontainer.json

```diff
@@ -29,7 +30,7 @@
   "forwardPorts": [
     4144, // TMS Backend
     5173, // TMS Frontend
-    5432  // Postgres Database
+    5432 // Postgres Database
```

Comment:
This is Prettier doing its thing. The IDE settings should have everyone set up to format on save, but maybe that isn't working?